src/sentry/api/endpoints/organization_sessions.py

```diff
@@ -23,8 +23,15 @@ def data_fn(offset: int, limit: int):
                 with sentry_sdk.start_span(
                     op="sessions.endpoint", description="build_sessions_query"
                 ):
+                    request_limit = None
```

Comment:
If no request limit or offset is sent, then I want to delegate the limit and offset setting logic to the query definition